app/assets/javascripts/mobile_tooltip.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+  /*
+  **  Tooltip for the profile page on mobile devices
+  **  This script should ideally run on the profile page
+
+  **  Behaviour
+      ========================
+      - Enables the viewers tooltip to be shown on mobile devices when the
+        parent element is clicked.
+      - Toggle behaviour is implemented so that the tooltip can be both shown
+        and dismissed
+
+  */
+
+  $(document).on("page:load ready", function () {
+    // Allow this script to only be run on the profile page
+
+    if (window.location.pathname !== "/profile") return false;
+
+    var toggleCount = 0;
+
+    // Ensure the script is activated on mobile devices only
+    // Max-width: 768px
+
+    if ($(window).width() <= 768) {
+      $(".some_focus_text").on("click", function () {
+        var child = $(this).children(".yes_title").eq(0),
+            title = child.prop("title");
+
+        child.tooltip({
+          content: title,
+          position: { my: "top-10 left+25", at: "top left", of: $(this) }
```

Comment:
Okay I had a tough time trying to figure out why the position of the tooltip was dead center instead of any sort of top left alignment.

<img width="305" alt="screen shot 2016-06-17 at 2 37 57 pm" src="https://cloud.githubusercontent.com/assets/1372018/16162716/c6b90064-349a-11e6-8a2d-04f7b54b85b0.png">

It turns out that these arguments need to be flipped (it's horizontal, vertical and not vertical, horizontal) so this would become 
```
position: { my: "left+25 top-10", at: "left top", of: $(this) }
```
which yields this position, which makes a lot more sense but IMO doesn't look great (kind of confusing as to which item we're talking about, I clicked on the 2nd one)

<img width="311" alt="screen shot 2016-06-17 at 2 51 29 pm" src="https://cloud.githubusercontent.com/assets/1372018/16162782/12df610e-349b-11e6-9dc1-b6fbeccb8a46.png">

I think based on how this looks on desktop, I'd like to propose

```
position: { my: "left+40 top+25", at: "left top" }
```
(note no more `of: $(this)`) which yields this

<img width="308" alt="screen shot 2016-06-17 at 2 53 56 pm" src="https://cloud.githubusercontent.com/assets/1372018/16162828/5e3eede0-349b-11e6-9e99-643eb6cae288.png">

Compare to desktop:

<img width="522" alt="screen shot 2016-06-17 at 2 54 50 pm" src="https://cloud.githubusercontent.com/assets/1372018/16162854/81a88a16-349b-11e6-8c9e-009483583f7d.png">

But of course, open to other suggestions as well.